### PR TITLE
Hooli-style-Popup-fix-bug

### PR DESCRIPTION
**Pull request by SkirnevskyiAleksandr**
###  Hooli-style-Popup

[Demo](https://skirnevskyialeksandr.github.io/Hooli-style-Popup/)
[Code base](https://github.com/SkirnevskyiAleksandr/Hooli-style-Popup)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

**Comment by SkirnevskyiAleksandr** [comment]
Hi, this is my second pull request, in the first one I made a mistake that I can't fix:
pull request mistake
so I'm making a new pull request. Your comments on my code from first PR I fixed.

**Comment by OleksiyRudenko** [comment]
This PR is a replacement for #239

File: submissions/SkirnevskyiAleksandr/HTML-CSS-Popup/index.html [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+  </head>
+
+  <body>
+    <!--header-->
+    <header class="container">
+      <!--header-menu-->
+      <nav class="menu">
+        <ul class="menu">
+          <li class="menu-item"><a class="menu-item" href="#">Mail</a></li>
+          <li class="menu-item"><a class="menu-item" href="#">Images</a></li>
```
Comment (A-Ostrovnyy) [inline submissions/SkirnevskyiAleksandr/HTML-CSS-Popup/index.html:17]: Fix `outline` on focus state. Same problem on icons inside the popup.
image

File: submissions/SkirnevskyiAleksandr/HTML-CSS-Popup/index.html [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+    <link rel="stylesheet" href="./style.css" />
+    <title>Document</title>
+  </head>
+
+  <body>
+    <!--header-->
+    <header class="container">
+      <!--header-menu-->
+      <nav class="menu">
+        <ul class="menu">
```
Comment (A-Ostrovnyy) [inline submissions/SkirnevskyiAleksandr/HTML-CSS-Popup/index.html:15]: Different by functionality elements with the same class name is a bad practice. Please fix it.

File: submissions/SkirnevskyiAleksandr/HTML-CSS-Popup/index.html [hunk 3]
```diff
@@ -0,0 +1,223 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./style.css" />
+    <title>Document</title>
```
Comment (A-Ostrovnyy) [inline submissions/SkirnevskyiAleksandr/HTML-CSS-Popup/index.html:7]: Add a propper page title.

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!
See my comments above!

File: submissions/SkirnevskyiAleksandr/HTML-CSS-Popup/index.html [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+            <input type="checkbox" id="drop-down-btn" class="drop-down-input" />
+            <label class="drop-down-input" for="drop-down-btn"
+              ><img
+                class="picture"
+                src="./icons/popup-button.png"
+                alt="popup-button"
+            /></label>
+            <!--dropdown-menu-->
+            <nav class="dropdown-container">
+              <ul class="drop-down-menu">
```
Comment (A-Ostrovnyy) [inline submissions/SkirnevskyiAleksandr/HTML-CSS-Popup/index.html:29]: Try to write class names or JS variables names in one style, It easier to read it in the future)

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
I see your changes in the demo but not in commits. Please add commit with changes to this repo.

**Review by A-Ostrovnyy — APPROVED** [review]
Well done!

**Pull request merged**